packages/host/src/Instance.js

```diff
@@ -284,9 +301,9 @@ class Instance extends libLink.Link {
 			instance_id: this.id,
 			type: "join",
 			name,
-			stats: stats.toJSON(),
+			stats,
 		};
-		libLink.messages.playerEvent.send(this, event);
+		this.sendTo(new libData.InstancePlayerUpdateEvent("join", name, undefined, stats), "controller");
```

Comment:
Same here on formatting, the second variant in this case is not really needed - I usually chunk that when it hits the 120 characters on the line - that's when the code line just gets a bit too long
```
this.sendTo(
    new libData.InstancePlayerUpdateEvent("join", name, undefined, stats), 
    "controller"
);
```
or
```
this.sendTo(
    new libData.InstancePlayerUpdateEvent(
        "join", 
        name, 
        undefined, 
        stats
    ), 
    "controller"
);
```

Reply:
Perhaps the argument order for sendTo should be reversed?
```js
this.sendTo("controller", new libData.InstancePlayerUpdateEvent("join", name, undefined, stats));
```

To me something like this also flows nicely when reading it.
```js
this.sendTo("controller",
    new libData.InstancePlayerUpdateEvent("join", name, undefined, stats)
);
```